### PR TITLE
update contact section of the html  the css styles of the contact se…

### PR DESCRIPTION
**Pull request by ANTOINE1128**
add  contact section in the HTML file
add style to the HTML contact section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
...
                     </section>
+                    <section class = contact-section> 
+                        <h3 class ="contact-description">
+                            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                        </h3>
+                        <ul class="input-filed">
+                        <li><label for="email"> <input type="email" id="email" name="email" placeholder="Full name"></label></li>
+                       <li> <label for="e-mail">
+                        <input type="text" name="e-mail" id="e-mail" placeholder="e-mail address"></label></li>  
+                     </ul>
```
Comment (AdnanOlarmmi) [inline index.html:171]: - [ ] Your form section should be in a `form` element, this is good for semantic and effectivity reasons, check [this article](https://social.msdn.microsoft.com/Forums/en-US/500e1609-612d-41a5-bb5b-30b83ee40b90/form-and-div-differences?forum=aspwebforms) for more.

- It (the form) should likewise have an `action` attribute leading to a FormSpree submission, and a `method` attribute, with the value `POST` to indicate that data is been submitted to the database.

Check out [how to use form spree here!](https://www.youtube.com/watch?v=gWgzzVVcqfA)

File: index.html [hunk 2]
```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
                             </div>
                         </div>
                     </section>
+                    <section class = contact-section> 
+                        <h3 class ="contact-description">
+                            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                        </h3>
+                        <ul class="input-filed">
+                        <li><label for="email"> <input type="email" id="email" name="email" placeholder="Full name"></label></li>
```
Comment (AdnanOlarmmi) [inline index.html:168]: - [ ] As it is in the requirement, all the input fields should have the `required` attribute, to ensure that proper information or data has been retrieved.

File: index.html [hunk 3]
```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
...
+                       <li> <label for="e-mail">
+                        <input type="text" name="e-mail" id="e-mail" placeholder="e-mail address"></label></li>
```
Comment (AdnanOlarmmi) [inline index.html:170]: - [ ] As it is in the requirement, all the input fields should have the `required` attribute, to ensure that proper information or data has been retrieved.

File: index.html [hunk 4]
```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
...
+                    <section class = contact-section> 
+                        <h3 class ="contact-description">
+                            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                        </h3>
+                        <ul class="input-filed">
+                        <li><label for="email"> <input type="email" id="email" name="email" placeholder="Full name"></label></li>
+                       <li> <label for="e-mail">
+                        <input type="text" name="e-mail" id="e-mail" placeholder="e-mail address"></label></li>  
+                     </ul>
+                       <textarea name="comment" id="comment" cols="30" rows="10">write me something</textarea>
```
Comment (AdnanOlarmmi) [inline index.html:172]: - [ ] As it is in the requirement, all the input fields should have the `required` attribute, to ensure that proper information or data has been retrieved.

Comment (AdnanOlarmmi) [inline index.html:172]: - [ ] The `textarea` element should also be in between the `form` element. 😉

File: index.html (hunk 2 above)
Comment (AdnanOlarmmi) [inline index.html:168]: - [ ] Kindly research and add the limitation for maximum length, as it is a requirement 😉.

File: index.html (hunk 4 above)
Comment (AdnanOlarmmi) [inline index.html:172]: - [ ] Kindly research and add the limitation for maximum length, as it is a requirement 😉.

File: index.html
Comment (AdnanOlarmmi) [inline index.html:1]: - As per the design. 

- What you submitted
image

- What is expected
right

- Kindly fix all aspect of the design to match the Figma design as much as possible, 🏃

File: index.html [hunk 5]
```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
...
+                       <button class="getintouch">get in touch<button>
```
Comment (AdnanOlarmmi) [inline index.html:173]: - [ ] Kindly ensure that your buttons also follow the design guidelines, such as `enabled`, `hover`, `pressed`, and `disabled`, so as to have a pixel-perfect design like you'll soon be doing in your job 🙂.

- Your design
mobile

- Figma design
button2

- And please remove the black border around the button, and fix the capitalization of the text in the button.

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @ANTOINE1128, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Keep Improving.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [x] I suggest that you take Google or other available search engines as your friend in your developer journey, as a developer *you aren't paid for what you know, but where you can find the answer*, please never give up. Most developers don't know this from the top of their head, but by researching, asking questions and making your finding, things will become more understandable. Please take this advice of mine as a little token from me to you. 😃 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ANTOINE1128** [comment]
- added the form section 
- added  an action attribute leading to a FormSpree submission, and a method attribute, with the value POST to indicate that data is been submitted to the database
- textarea element is also in between the form element.
- As it is in the requirement, all the input fields have the required attribute
- add the limitation for maximum length
-  fixed all aspect of the design to match the Figma design as much as possible
-  removed the black border around the button, and fix the capitalization of the text in the button.
- the buttons also follow the design guidelines, such as enabled, hover, pressed, and disabled

File: index.html [hunk 6]
```diff
@@ -157,6 +160,32 @@ <h1 class="title">About me</h1>
...
+                        <label for="username"></label>
+                        <input type="text" id="username" name="username" placeholder="username" class="input" required maxlength="30">
+                        <br>
+                        <label for="email"></label>
+                        <input type="email" id="email" name="email" placeholder="email" class="input" required pattern=".*[.com]">
+                        <br><br>
+                        <textarea name="comment" id="comment" cols="30" rows="10" placeholder="write me something..." required maxlength="500"></textarea>
+                        <br>
+                        <label for="getintouch"></label>
+                        <input type="submit"  value="Get in touch" id="getintouch">
```
Comment (AdnanOlarmmi) [inline index.html:175]: - [ ] Please kindly ensure that the button matches the given Interactions on Figma.

- Your design
mobile

- Figma design
button2

- And it seems you are confusing clicked to disable 😃

File: index.html [hunk 7]
```diff
@@ -157,6 +160,32 @@ <h1 class="title">About me</h1>
...
+                    <form action="https://formspree.io/f/moqzkbjd" method="post">
+
+                        <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+                        <label for="username"></label>
+                        <input type="text" id="username" name="username" placeholder="username" class="input" required maxlength="30">
+                        <br>
+                        <label for="email"></label>
+                        <input type="email" id="email" name="email" placeholder="email" class="input" required pattern=".*[.com]">
+                        <br><br>
+                        <textarea name="comment" id="comment" cols="30" rows="10" placeholder="write me something..." required maxlength="500"></textarea>
```
Comment (AdnanOlarmmi) [inline index.html:172]: - [ ] Kindly remove the little black border around the `textarea` to perfectly match the design.

image

Comment (AdnanOlarmmi) [inline index.html:172]: - [ ] Kindly add the left margins to the input fields to match the design.

- Your design
image

- Figma design
image

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @ANTOINE1128, 

Hello there again 😃
This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Nice one using FormSpree.
- Nice design so far.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — APPROVED** [review]
Hi @ANTOINE1128, @gisachris  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Implemented changes requested by the previous reviewer💯
- Ui matches the Figma design 👍
- Nice and clean code 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**